zone/client.cpp

```diff
@@ -1100,6 +1100,7 @@ bool Client::Save(uint8 iCommitNow) {
 	database.SaveCharacterData(this, &m_pp, &m_epp); /* Save Character Data */
 
 	database.SaveCharacterEXPModifier(this);
+	database.SaveCharacterTitleSets(this);
```

Comment:
This PR looks great, generally I want us to be moving away from shoving so many things into Client::Save and actually work to save things when we need to save them selectively rather than brute-forcing saving a ton of stuff all of the time. 

Is there a better place where we could put this ?